other-guides/hosting-on-linux-server

```diff
@@ -0,0 +1,53 @@
+# Hosting on a Linux Server
+
+The best way to host your bot is on a Linux Server (or simply said: just a virtual server. Its also for bare metal servers).
+
+## Requirements
+
+- A Linux Server with a stable version, node, npm, git, pm2. 
+- Github repository from your bot (privat).
+- SSH Client
+
+## Getting Started
+
+### Login
+
+The best way to do that is with root access. If its root access the username would be `root`.
+The first step you do is to login into your Linux Server. You can use a SSH Client, with Windows CMD `ssh (username)@(ip)` or on macOS ssh 192.168.2.30 -l (username)`
+
+### Get Bot Files
+
+After you login in, clone your bot repository to your server. We use git for that.
+To clone your repository just type `git clone https://github.com/(username)/(rep-name).git`. Username is your github account name and rep-name the name of your repository.
+Now you need to go the file folder. Type `ls`to see dir of your server. If you found a folder with the name of the repository this should be your bot files folder.
+Now type `cd (folder-name)`to get there. Now type again `ls`. If you see the files from the bot your right. 
+
+### Starting Bot
+
+Now deleted the node_modules folder (if you have) with `rm -r node_modules`
+and type next `npm i`. You need to reinstall that because the Linux got a other compiler then your windows computer (this is because there are diffrent OS). After you installed node_modules,
```

Comment:
You should never commit `node_modules`, in fact, you must add it to `.gitignore`.